packages/metasploit-framework/build.sh

```diff
@@ -0,0 +1,44 @@
+TERMUX_PKG_HOMEPAGE=https://www.metasploit.com/
+TERMUX_PKG_DESCRIPTION="framework for pentesting"
+TERMUX_PKG_VERSION=4.16.2
+# Depend on coreutils for bin/env
+TERMUX_PKG_DEPENDS="wget, ruby, curl, autoconf, bison, clang, coreutils, curl, findutils, git, libffi-dev, libgmp-dev, libpcap-dev, postgresql-dev, readline-dev, libsqlite-dev, openssl-dev, libtool, libxml2-dev, libxslt-dev, ncurses-dev, pkg-config, postgresql-contrib, wget, make, ruby-dev, libgrpc-dev, termux-tools, ncurses-utils, ncurses, unzip, zip, tar, autoconf "
+TERMUX_PKG_PLATFORM_INDEPENDENT=no
+
+termux_step_post_massage() {
+	
+	
+	
+
+        cd $PREFIX/share/
+        apt update
+        cd $PREFIX/share/
+        curl -LO https://github.com/rapid7/metasploit-framework/archive/4.16.2.tar.gz
+        tar -xf $PREFIX/share/4.16.4.tar.gz
```

Comment:
You can use TERMUX_PKG_SRCURL instead of hardcoding the path here and manually downloading it.
Look at any other package to see how TERMUX_PKG_SRCURL+TERMUX_PKG_VERSION is used.

Reply:
And note that you download 4.16.2 but then try to extract 4.16.4